src/lib/elput/Elput.h

```diff
@@ -125,17 +125,17 @@ typedef struct _Elput_Event_Switch
 } Elput_Event_Switch;
 
 
-EAPI extern int ELPUT_EVENT_SEAT_CAPS;
```

Comment:
`Elput` isn't compiled at windows, do we really need to configure `EXTERN` here?

Reply:
You are right. Elput and Eldbus changes are really not necessary as they will never be part of win32 build. 12cbffc5a4